### DISCUSSION TITLE
Stopping production in the buildings

### DISCUSSION THREAD

**Discussion by Joahim49** [fedorovvl/tso_client#304] [Ideas]
Hey, will it be possible to stop and start in all buildings of a certain type of production at once?
For example: All lumberjacks stop ;) or all buildings stop production ;)

Warm greetings to the magicians, you are great ;) thanks ;)

**Comment by fedorovvl** [comment]
hi
u can do it in Production window
pls check wiki https://github.com/fedorovvl/tso_client/wiki/Main-info#production-f7

**Reply by Joahim49** [reply]
wow, thanks for the quick reply ;) I'm going to start reading, thx ;)
